Sources/Apollo/NormalizedCache.swift

```diff
@@ -1,5 +1,19 @@
 import Foundation
 
+/// An enumeration of behaviors for clear operations on a given cache.
+public enum CacheClearingPolicy: Equatable {
+  /// Clears all records in the cache.
+  case allRecords
+  /// Clears all records whose key matches the provided glob pattern.
+  ///
+  /// For example `*pollo` will match both `Apollo` and `pollo`.
+  case allMatchingKeyPattern(String)
+  /// Clears the the first (oldest) records in the cache up to the given limit.
+  case first(Int)
```

Comment:
Hmmm, I don't think we currently have a way to support the `first(Int)` and `last(Int)` policies. The implementations below use an unspecified order (for the in memory cache) or orthographic order of the key (for the SQLite cache), and neither maps to oldest or most recent. This probably has to wait until we have resolved https://github.com/apollographql/apollo-ios/pull/1498.

Reply:
Ah, you're right between this feedback and the one on Dictionary key ordering.

I'm just going to remove these policies. They can be re-added later if devs need them